event.c

```diff
@@ -383,7 +388,11 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
             return;
         }
     } else {
-        snprintf(fullfilename, PATH_MAX, "%s/lastsnap.%s", cnt->conf.filepath, imageext(cnt));
+		char filename[PATH_MAX];
```

Comment:
I was just trying to reduce the changes to existing code, I should close
this and recreate with the whitspace issues resolved (I blame geany :) ) as
well as moving the declarations up so no duplicates.

*Jim Dodgen*







On Sat, May 16, 2015 at 1:52 PM, tosiara <notifications@github.com> wrote:

> In event.c
> <https://github.com/Mr-Dave/motion/pull/73#discussion_r30464577>:
>
> > @@ -383,7 +388,11 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
> >              return;
> >          }
> >      } else {
> > -        snprintf(fullfilename, PATH_MAX, "%s/lastsnap.%s", cnt->conf.filepath, imageext(cnt));
> > +		char filename[PATH_MAX];
>
> So you are declaring those once again just because they are outside of
> scope? I thing doubling variables of the same name is not a good approach
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mr-Dave/motion/pull/73/files#r30464577>.
>